packages/twenty-front/src/pages/onboarding/SyncEmails.tsx

```diff
@@ -47,13 +58,13 @@ export const SyncEmails = () => {
   const [skipSyncEmailOnboardingStatusMutation] =
     useSkipSyncEmailOnboardingStepMutation();
 
-  const handleButtonClick = async () => {
+  const handleButtonClick = async (provider: string) => {
```

Comment:
Can we use a string literal instead of a string ? 

type EmailProvider = "google" | "microsoft" ...

Reply:
Smart